dashboard/modules/job/common.py

```diff
@@ -58,7 +62,8 @@ class JobStatusStorageClient:
     Handles formatting of status storage key given job id.
     """
 
-    JOB_STATUS_KEY = "_ray_internal_job_status_{job_id}"
+    JOB_STATUS_KEY_PREFIX = "_ray_internal_job_status"
+    JOB_STATUS_KEY = f"{JOB_STATUS_KEY_PREFIX}_{{job_id}}"
```

Comment:
nested f string 🤯

Reply:
haha yeah, it's a little ugly but the best I could come up with...